test/TestUtils.py

```diff
@@ -154,3 +155,15 @@ def test_normpath_with_string(self):
         self.assertEqual(
             utils.normpath("a/b/../"),
             "a")
+
+    def test_expand_path_vars(self):
```

Comment:
Since this test suite doesn't share a state on the class level, please make it a normal pytest test function:
```suggestion
def test_expand_path_vars():
```